### DISCUSSION TITLE
What's New in v0.2.0

### DISCUSSION THREAD

**Discussion by green-dalii** [green-dalii/diflowy#10] [Announcements]
![Operation_0 2 0](https://github.com/user-attachments/assets/1170735d-52cd-4133-bc8a-17a62d1c71ad)

# Multi-User Collaboration in Workspaces

To better meet the advanced needs of team and enterprise users, we are thrilled to announce that Diflowy now supports **multi-user collaboration in Workspaces**. This means you can share and collaborate on multiple versions of workflows within your workspace, facilitating efficient communication of Dify Workflow files with your team members.

For more information, please refer to: [Our Blog](https://diflowy.greenerai.top/blog/diflowy-workspace-multi-user-collaboration-beta-launch-en)

# How to experience it?

Currently, the Workspace collaboration feature is an advanced capability in beta testing. We welcome you to apply for beta access. 

If you would like to experience this feature, you can send an email to **diflowy@greenerai.top** or comment below with your registered **username** (when logging in with Github) or **registered account email** (when logging in with Google or email) stating your requirements and we will review and approve.

If you want to speed up the application process, we welcome you to upload more high-quality Dify Workflow to Diflowy or share your experience with your friends on social media using the hashtag “_**#Diflowy**_”.

**Comment by linis2000** [comment]
个人内测申请

**Reply by green-dalii** [reply]
你的注册账户就使用的@linis2000吧？我已经升级你的Plan，请查收，祝用的开心
